docs/pages/deploy/send-over-the-air-updates.mdx

```diff
@@ -26,3 +26,7 @@ To send an update, run the following [EAS CLI](/develop/tools/#eas-cli) command:
 This command will create an update and make it available to builds of your app that are configured to receive updates on the `production` channel. This channel is defined in [**eas.json**](/eas/json/#channel).
 
 You can verify the update works by force closing the app and reopening it two times. The update should be applied on the second launch.
+
+## Learn more
```

Comment:
one thing we need to work out here is - how should this page relate to [the update introduction page](https://docs.expo.dev/eas-update/introduction/)? (@quinlanj has a pr open to improve that one too: https://github.com/expo/expo/pull/31669)

Reply:
I'm not sure what I can do to resolve this. Do you have more specifics on what I need to change or talk to Quin about? This PR links to the update introduction with the [update guides]() link in the Learn more section.